string move_name
string dmp_name
float32[] goal
float32 time_to_execute
